tibber/__init__.py

```diff
@@ -480,11 +524,15 @@ async def update_info_and_price_info(self):
             % self._home_id
         )
 
-        self.info = await self._tibber_control.execute(query)
-        self._process_price_info(self.info)
+        if data := await self._tibber_control.execute(query):
+            self.info = data
+            self._process_price_info(self.info)
+        else:
+            # TODO: Error handling?
```

Comment:
Do you think it's okay to just return None instead of overwriting self.info with a potential None value? Does this break any functionality? There are 2 other places where I've done this (3 in total)

Reply:
@Danielhiversen ^

Reply:
return None is ok for this pr